app/components/footer.tsx

```diff
@@ -63,7 +63,7 @@ function ContactSection() {
       <ul className="mt-4">
         <FooterLink name="contact page" href="/contact" />
         <FooterLink name="office hours" href="/office-hours" />
-        <FooterLink name="Join discord" href="https://kcd.im/discord" />
+        <FooterLink name="Join discord" href="/discord" />
```

Comment:
Discord is also mentioned under `sitemap`. Should it be listed twice?

Reply:
Yeah, let's replace this with:

```suggestion
        <FooterLink name="Call Kent" href="/call" />
```